src/services/textChanges.ts

```diff
@@ -1103,15 +1138,23 @@ namespace ts.textChanges {
                     deleteImportBinding(changes, sourceFile, node as NamespaceImport);
                     break;
 
+                case SyntaxKind.SemicolonToken:

```

Comment:
I could be miscounting the scopes, but why would `deleteDeclaration` be called on a semicolon?

Reply:
`convertToEs6Module` deletes semicolons when changing an assignment statement to a function declaration.